addons/project_timesheet_holidays/models/hr_employee.py

```diff
@@ -7,29 +7,30 @@
 class Employee(models.Model):
     _inherit = 'hr.employee'
 
-    @api.model
-    def create(self, vals):
-        result = super(Employee, self).create(vals)
+    @api.model_create_multi
+    def create(self, vals_list):
+        employees = super().create(vals_list)
 
         # We need to create timesheet entries for the global time off that are already created
         # and are planned for after this employee creation date
 
         # First we look for the global time off that are already planned after today
         today = fields.Datetime.today()
-        global_leaves = result.resource_calendar_id.global_leave_ids.filtered(lambda l: l.date_from >= today)
-        work_hours_data = global_leaves._work_time_per_day()
+        for employee in employees:
```

Comment:
We can call the create only once `part 1 out of 3`
```suggestion
        lines_vals = []
        for employee in employees:
```